File: src/main/java/load/objects/AWSURI.java
package load.objects;

public class AWSURI {
    private String method;
    private String serviceName;
    private String regionName;
    private String host;
    private String canonicalUri;
    private String canonicalQueryString;
    private String fullURL;

    @Override
    public String toString() {
        return "AWSURI{" +
                "method='" + method + '\'' +
                ", serviceName='" + serviceName + '\'' +
                ", regionName='" + regionName + '\'' +
                ", host='" + host + '\'' +
                ", canonicalUri='" + canonicalUri + '\'' +
                ", canonicalQueryString='" + canonicalQueryString + '\'' +
                ", fullURL='" + fullURL + '\'' +
                '}';
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public String getServiceName() {
        return serviceName;
    }

    public void setServiceName(String serviceName) {
        this.serviceName = serviceName;
    }

    public String getRegionName() {
        return regionName;
    }

    public void setRegionName(String regionName) {
        this.regionName = regionName;
    }

    public String getHost() {
        return host;
    }

    public void setHost(String host) {
        this.host = host;
    }

    public String getCanonicalUri() {
        return canonicalUri;
    }

    public void setCanonicalUri(String canonicalUri) {
        this.canonicalUri = canonicalUri;
    }

    public String getCanonicalQueryString() {
        return canonicalQueryString;
    }

    public void setCanonicalQueryString(String canonicalQueryString) {
        this.canonicalQueryString = canonicalQueryString;
    }

    public String getFullURL() {
        return fullURL;
    }

    public void setFullURL(String fullURL) {
        this.fullURL = fullURL;
    }
}

File: src/main/java/load/gatling/scala/AWSURI.scala
class AWSURI {
  private var method: String = null
  private var serviceName: String = null
  private var regionName: String = null
  private var host: String = null
  private var canonicalUri: String = null
  private var canonicalQueryString: String = null
  private var fullURL: String = null

  override def toString: String = {
    return "AWSURI{" + "method='" + method + '\'' + ", serviceName='" + serviceName + '\'' + ", regionName='" + regionName + '\'' + ", host='" + host + '\'' + ", canonicalUri='" + canonicalUri + '\'' + ", canonicalQueryString='" + canonicalQueryString + '\'' + ", fullURL='" + fullURL + '\'' + '}'
  }

  def getMethod: String = {
    return method
  }

  def setMethod(method: String) {
    this.method = method
  }

  def getServiceName: String = {
    return serviceName
  }

  def setServiceName(serviceName: String) {
    this.serviceName = serviceName
  }

  def getRegionName: String = {
    return regionName
  }

  def setRegionName(regionName: String) {
    this.regionName = regionName
  }

  def getHost: String = {
    return host
  }

  def setHost(host: String) {
    this.host = host
  }

  def getCanonicalUri: String = {
    return canonicalUri
  }

  def setCanonicalUri(canonicalUri: String) {
    this.canonicalUri = canonicalUri
  }

  def getCanonicalQueryString: String = {
    return canonicalQueryString
  }

  def setCanonicalQueryString(canonicalQueryString: String) {
    this.canonicalQueryString = canonicalQueryString
  }

  def getFullURL: String = {
    return fullURL
  }

  def setFullURL(fullURL: String) {
    this.fullURL = fullURL
  }
}